desktop/main.ts

```diff
@@ -420,14 +428,13 @@ const mainWindow = () => {
                             },
                             {
                                 id: 'forward',
-                                role: 'forward',
```

Comment:
`forward` role doesn't exist, and overall since the menu item has click function it wasn't necessary. See more in [docs](https://www.electronjs.org/docs/latest/api/menu-item#roles)